docs/getting-started.md

```diff
@@ -50,6 +50,14 @@ For brand-new learners of Polkadot:
 For brand-new learners of Kusama: To learn more about how to build and maintain on the Kusama
 network, please head over to our [Kusama Guide][kusama guide].
 
+## Resources
+
+- [Community / Ecosystem](community) - List of community rooms and channels to talk to others about
+  Polkadot.
+- [Sample Applications](build-examples-index) - Sample applications that are built on or currently
+  being built for Polkadot.
+- [Contributing Guide](contributing) - Rules for contributing to the wiki.
+
```

Comment:
If anyone wants to see more content from the wiki index added on this page, let me know! :)